### ISSUE TITLE
[Proposal] Pull rxvc into rxv

### ISSUE THREAD

**Issue by wuub** [Raynes/rxvc#1]
Hi @Raynes,

Would you be against pulling rxvc's code right into rxv to be maintained under rxv itself?  (with proper attribution of course) ? We've added some stuff recently regarding zones and I would like to be able to iterate both on internal lib and cli at the same time.

**Comment by Raynes** [comment]
Certainly! I'd be honored actually. That library made what I've done incredibly useful.
